macros/sql/get_column_values.sql

```diff
@@ -18,9 +18,7 @@ Returns:
     {% endif %}
 {#--  #}
 
-    {%- set target_relation = adapter.get_relation(database=table.database,
-                                          schema=table.schema,
-                                         identifier=table.identifier) -%}
+    {%- set target_relation = table -%}
```

Comment:
Calling `get_relation` here serves a purpose - it checks to see if the target relation actually exists in the database. If it doesn't exist, then `target_relation` will be `none` and this macro will return the `default`.

This can happen if you're doing `dbt compile` on a fresh project, for instance. Previously, the query in this macro would fail -- dbt would try to select from a nonexistent model.

I think ideally, we'd have a different adapter method which checks that a given Relation exists. Today though, we need to pull apart the Relation to pass it into `get_relation`, which is pretty weird!

Reply:
Ah got it! I'll revert this commit then